xdsl/utils/lexer.py

```diff
@@ -112,6 +117,7 @@ def print_with_context(self, msg: str | None = None) -> str:
         are highlighted by up-carets beneath them (`^`). The message msg is printed
         along these.
         """
+        print("line_offset:", self.line_offset)
```

Comment:
Do we want to capture the output here?

Reply:
How did this not trigger anything :exploding_head: Thanks!